kinematics_simulator/SingleMinco[] trajs
